src/lib/amplitude/types.ts

```diff
@@ -67,6 +67,9 @@ export enum AmpEvent {
   TO_PUBLISH_MODULE = "To Publish Module",
   TO_DEPLOY_SCRIPT = "To Deploy Script",
   TO_MY_PUBLISHED_MODULES = "To My Published Modules",
+  TO_NFT_COLLECTIONS = "To NFT Collections",
+  TO_NFT_COLLECTION_DETAIL = "To NFT Collection Detail",
```

Comment:
Detail or Details?

Reply:
If you mean whether "Detail" or "Details" is correct, then "Details" seems more accurate.
i will change that.

Reply:
am I understanding this correctly? Most value use "Detail".